tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -18,41 +15,105 @@ ${pachyderm_appname}            pachyderm
 
 
 *** Test Cases ***
-Verify Pachyderm Can Be Installed Using OpenShift Console
-    [Documentation]     Check if it is possible to install and deploy pachyderm server successfully.
+Verify Pachyderm Can Be Installed And Deployed
+    [Documentation]     Check if it is possible to install and deploy pachyderm.
     [Tags]      Tier2
     ...         ODS-1137    ODS-1138
-    Check And Install Operator in Openshift    ${pachyderm_container_name}    ${pachyderm_appname}
-    Create Project      pachyderm
-    Create Pachyderm AWS-Secret
-    Create Tabname Instance For Installed Operator        ${pachyderm_container_name}   ${pachyderm_container_name}     pachyderm
-    Wait Until Status Is Running
+    Pass Execution      Passing test, as suite setup ensures Pachyderm operator is installed correctly.
+
+Verify Pachyderm Pipeline Can Be Created
+    [Documentation]     Checks if it is possible to create sample pipline using jupyterlab.
+    [Tags]      Tier2
+    ...         ODS-1161
+    Go To    ${OCP_CONSOLE_URL}
+    ${pachctl_version}=     Get Pachd Version
     Go To RHODS Dashboard
     Verify Service Is Enabled    Pachyderm
-    [Teardown]  Uninstall Pachyderm Operator
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook With Arguments  image=s2i-generic-data-science-notebook
+    Create Pachyderm Pipeline Using JupyterLab     ${pachctl_version}
+    Verify Pipeline Pod Creation
+    [Teardown]  Delete Pipeline And Stop JupyterLab Server
 
 *** Keywords ***
-Pachyderm Test Suite
-    Set Library Search Order  SeleniumLibrary
-    RHOSi Setup
-
-Dashboard Test Setup
+Pachyderm Suite Setup
     Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
     Open Browser    ${OCP_CONSOLE_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
     Login to OCP
     Wait Until OpenShift Console Is Loaded
+    Check And Install Operator in Openshift    ${pachyderm_container_name}    ${pachyderm_appname}
+    Create Project      pachyderm
+    Create Pachyderm AWS-Secret
+    Create Tabname Instance For Installed Operator        ${pachyderm_container_name}   ${pachyderm_container_name}     ${pachyderm_appname}
+    Wait Until Status Is Running
 
-Dashboard Test Teardown
+Pachyderm Suite Teardown
```

Comment:
the teardown should close all the browser

Reply:
Pachyderm Suite Teardown has an implementation of closing all browsers.

Reply:
oh sorry, I missed it. you're right